lib/resty/balancer/utils.lua

```diff
@@ -11,27 +11,10 @@ do
         new_tab = function (narr, nrec) return {} end
     end
 end
-
 _M.new_tab = new_tab
 
-local copy
-do
-    local ok
-    ok, copy = pcall(require, "table.clone")
-    if not ok or type(copy) ~= "function" then
-        copy = function(nodes)
-            local newnodes = new_tab(0, 0)
-            for id, weight in pairs(nodes) do
-                newnodes[id] = weight
-            end
-
-            return newnodes
-        end
-    end
-end
-_M.copy = copy
 
-local nkeys
+local nkeys, jit_nkeys
```

Comment:
It's confusing to have both `nkeys` and `jit_nkeys`.

Reply:
how about `nkeys` and `raw_jit_nkeys`?

Reply:
what is the difference here? It's still confusing.